js/client/modules/@arangodb/aql-helper.js

```diff
@@ -458,21 +447,67 @@ function unpackRawExpression (node, transform = false) {
 function sanitizeStats (stats) {
   // remove these members from the stats because they don't matter
   // for the comparisons
-  delete stats.scannedFull;
-  delete stats.scannedIndex;
-  delete stats.cursorsCreated;
-  delete stats.cursorsRearmed;
-  delete stats.cacheHits;
-  delete stats.cacheMisses;
-  delete stats.filtered;
-  delete stats.executionTime;
-  delete stats.httpRequests;
-  delete stats.fullCount;
-  delete stats.peakMemoryUsage;
-  delete stats.intermediateCommits;
+  [
+    'scannedFull',
+    'scannedIndex',
+    'cursorsCreated',
+    'cursorsRearmed',
+    'cacheHits',
+    'cacheMisses',
+    'filtered',
+    'executionTime',
+    'httpRequests',
+    'fullCount',
+    'peakMemoryUsage',
+    'intermediateCommits'].forEach(item => { delete stats[item]; });
   return stats;
 }
 
+function execute_query(query, bindVars = null, options = {}) {
+  let stmt = db._createStatement({query, bindVars: bindVars, count: true});
+  return stmt.execute();
+};
+
+function execute_all_json(plans, result, query) {
+  plans.forEach(function (plan) {
+    var jsonResult = db._executeJson(plan, {optimizer: {rules: ['-all']}});
+    assertEqual(jsonResult, result, query);
+  });
+};
+
+function query_cache_properties(arg) {
+  let command;
+  if (arg) {
+    command = `
+      return AQL_QUERY_CACHE_PROPERTIES(${JSON.stringify(arg)})
+    `;
+  } else {
+    command = `
+      return AQL_QUERY_CACHE_PROPERTIES()
+    `;
+  }
+
+  let res = arango.POST("/_admin/execute", command);
+  return res;
```

Comment:
This still relies on JavaScript running inside the server.
It should instead call the REST API `GET /_api/query/properties`.